R/get_functions.R

```diff
@@ -42,6 +43,7 @@ get_estimates <- function(design, data = NULL, start = 1, end = length(design))
 #' @rdname get_functions
 #' @export
 draw_assignment <- function(design, data = NULL, start = 1, end = length(design)) {
+  check_design_class_single(design)
```

Comment:
It's already in get_function_internal, so you don't need it in every single wrapper.

Reply:
thx